x/token/keeper/keeper_test.go

```diff
@@ -78,7 +78,7 @@ func (s *KeeperTestSuite) SetupTest() {
 		&s.customer,
 		&s.stranger,
 	}
-	for i, address := range s.createRandomAccounts(len(addresses)) {
+	for i, address := range createRandomAccounts(len(addresses), s.deterministic) {
```

Comment:
I don't think this change is necessary, because `TestMsgMintVerifyEvent` already use expected variable event result. Isn't it?

Reply:
I just reverted it.